openlibrary/templates/search/lists.html

```diff
@@ -1,30 +1,53 @@
-$def with (q, lists)
+$def with (get_results)
```

Comment:
Instead of passing forward the function `get_results`, call `get_results` in the python code and pass forward the response. This lets the template focus just on rendering.